src/main/java/proto/Pose.proto

```diff
@@ -0,0 +1,20 @@
+/*
+ * This proto format is used along with the delimited methods to
+ * store message size for easy decoding/encoding.
+ */
+syntax = "proto3";
+
+package formats;
+option java_package = "proto";
+option java_outer_classname = "SensorData";
+
+message Pose{
+  // Position is in WGS84 latitude, longitude, altitude.
+  double gps_timestamp = 1; // Time in seconds.
+  double lat = 2;  // Degrees in [-90, 90].
+  double lng = 3;  // Degrees in [-180,180].
+  double alt = 4; // Meters
+  float roll_deg = 5; // Degrees per second
+  float pitch_deg = 6; // Degrees per second
+  float yaw_deg = 7; // Degrees per second
```

Comment:
I don't think these are supposed to be angular speeds.

Reply:
Whoops

Reply:
This should be degrees.